book/src/development/the_team.md

```diff
@@ -0,0 +1,112 @@
+    subscribe to the issue and help interested parties, if they post questions
+    in the comments. 
+
+2. **Closing duplicate or resolved issues**
+
+    When you manually close and issue, it's often a good idea, to add a short
+    comment explaining the reason.
+
+3. **Ping people after two weeks of inactivity**
+
+    We try to keep issue assignments and PRs fairly up-to-date. After two weeks,
+    it can be good to send a friendly ping to the delaying party.
+
+    You might close a PR with the `I-inactive-closed` label if the author is
+    busy or wants to abandon it. If the reviewer is busy, the PR can be
+    reassigned to someone else.
+
+    Checkout: https://triage.rust-lang.org/triage/rust-lang/rust-clippy to
+    monitor PRs.
+
+### Membership
+
+If you have been contributing to Clippy for some time, we'll probably ask you if
+you want to join this team. Members of this team are also welcome to suggest
+people who they think would make a great addition to this group.
+
+For this group, there is no direct onboarding process. You're welcome to just
+continue what you've been doing. If you like, you can ask for someone to mentor
+you, either in the Clippy stream on Zulip or privately via a PM.
+
+If you have been inactive in Clippy for over three months, we'll probably move
+you to the alumni group. You're always welcome to come back.
+
+## The Clippy Team
+
+[The Clippy team](https://www.rust-lang.org/governance/teams/dev-tools#Clippy%20team)
+is responsible for maintaining Clippy.
+
+### Duties
+
+1. **Respond to PRs in a timely manner**
+
+    It's totally fine, if you don't have the time for reviews right now.
+    You can reassign the PR to a random member by commenting `r? clippy`.
+
+2. **Take a break when you need one.**
+
+    You are valuable! Clippy wouldn't be what it is without you. So take a break
+    early and recharge some energy when you need to.
+
+3. **Sync Clippy with the rust-lang/rust repo**
+
+    This should be done roughly every two weeks. This is usually done by our
+    king @flip1995.
+
+4. **Update the changelog**
+
+    This needs to be done for every release, every six weeks. This is usually
+    done by @xFrednet.
+
+### Membership
+
+If you have been in the *Clippy-Contributors* team for some time, we'll probably
+reach out and ask if you want to help with reviews and eventually join the
+Clippy team.
+
+During the onboarding process, you'll have an active Clippy team member as a
+mentor, who assigns PRs to you. They will shadow your reviews, meaning that
+they'll keep an eye on your PRs, help you with any questions, and once you're
+done, perform a full review. When everything looks good, they'll r+ the PR in
+the name of both of you.
+
+When you've done several reviews and seem confident in the role, you'll be
+invited to join the team officially, as long as you're still interested. This
+onboarding phase typically takes a couple of weeks to a few months.
```

Comment:
```suggestion
When you've done several reviews and you and your mentor seem confident about becoming a member, you'll be
invited to join the team officially, as long as you're still interested. This
onboarding phase typically takes a couple of weeks to a few months.
```

Or something like this. You have a better way with words than me.

I would also mention, that there will be an informal "vote" by the current members about them joining. Maybe also that when the mentor thinks they're ready, the vote is usually unanimous.

Reply:
```suggestion
When you've done several reviews, and you and your mentor seem confident having you as a member, you'll be
invited to join the team officially if you're still interested. This
onboarding phase typically takes a couple of weeks to a few months.

Reply:
It might help, that I'm currently working on my master thesis. Reformulating the same paragraph over and over is... an experience :D

This will be updated in the next commit